hrjobcontract/js/src/job-contract/controllers/form/form-general.js

```diff
@@ -89,7 +89,7 @@ define([
         months = m.diff(dateStart, 'months');
 
         m.add(-months, 'months');
-        days = m.diff(dateStart, 'days');
+        days = m.diff(dateStart, 'days') + 1;
```

Comment:
I played around a while with moment.js humanize method, but it really doesn't do what we'd need it to.  Even tried to humanize each part of the date (years, months, days) independently, but strange things happened on frontier cases, like a duration of 31 days, for example which got rendered as 30 days and a day.  You can use Relative Time Thresholds to set the value where a set of days gets rounded to a month, but you'd have to start to juggle it before and after you call humanize for months and days, which is kind of awkward.  There's a plugin for moment.js that does what we need, but it basically does the same thing our method does, so I don't know if it's woth it. http://momentjs.com/docs/#/plugins/preciserange/